config/default/foia_wizard.settings.yml

```diff
@@ -156,7 +156,7 @@ messages:
     value: "<p>These records appear to be records of a state or local authority.&nbsp; If you are looking for non-federal records, such as state or local records, we suggest that you contact the proper state or local authority in accordance with the appropriate state or local records access law.</p>\r\n"
     format: rich_text_wizard
   m50:
-    value: "<p>Message 50</p>\r\n"
+    value: "<p>For all other records, we recommend you contact your medical provider directly</p>\r\n"
     format: rich_text_wizard
   m51:
```

Comment:
We'll use the same message as m50 here.

Reply:
I just noticed the pronoun difference on this one (`your` vs. `the`), I'll update the front-end too to use this one when it's for someone else's records